$NetBSD: patch-aa,v 1.2 2010/01/23 04:50:22 obache Exp $

* getaddrinfo(3) is in XNS5.2, require _XOPEN_SOURCE>=520
* _NETBSD_SORCE is required for NI_MAXHOST/NI_MAXSERVE on NetBSD.

--- dbmail.h.in.orig	2010-01-22 12:57:30.000000000 +0000
+++ dbmail.h.in
@@ -33,7 +33,10 @@
 #ifdef __FreeBSD__
 #define _XOPEN_SOURCE
 #else
-#define _XOPEN_SOURCE 	500
+#define _XOPEN_SOURCE 	520
+#endif
+#ifdef __NetBSD__
+#define _NETBSD_SOURCE
 #endif
 
 #define __EXTENSIONS__ 		/* solaris */
